lib/fog/ovirt/requests/compute/v4/add_volume.rb

```diff
@@ -32,12 +32,12 @@ def add_options_defaults(options)
             options[:disk] ||= {}
             options[:disk][:storage_domains] ||= [client.system_service.storage_domains_service.storage_domain_service(options[:storage_domain_id]).get]
             options[:disk][:provisioned_size] ||= options.delete(:provisioned_size)
-            options[:disk][:type] ||= options.delete(:type)
```

Comment:
Is `type` really not needed anymore?

Reply:
it seems that it's not needed..

Reply:
ovirt api v4 removed the type from disk: http://ovirt.github.io/ovirt-engine-api-model/4.2/#_remove_the_disk_code_type_code_property